### ISSUE TITLE
Add support for nosql like mongo

### ISSUE THREAD

**Issue by John-H-Smith** [LogBlock/LogBlock#838]
As for large servers, there are extreme large databases which can produce a very very slow performance.
Nosql, for example mongodb, is mainly used for storing big data.
It would be a very great enhancement if you could serve the option to use mysql OR mongodb.

**Comment by Brokkonaut** [comment]
I don't think that using mongodb would improve the performance. and it would be a lot of work to implement that feature, so i wouldn't expect it. but if someone wants to send a pr..

**Comment by Noorquacker** [comment]
https://stackoverflow.com/questions/9702643/mysql-vs-mongodb-1000-reads

Please, no trendy technology buzzword stuff in a barely-maintained Minecraft server plugin. Per the link above, MongoDB is not magically faster for "big data", it's faster if you really plan ahead and tailor-fit it for MongoDB. If it runs fine in MySQL, it's not going to be faster in MongoDB.
